### DISCUSSION TITLE
[Support] 一体化项目下，开发环境 mwtsc 启动 bootstrap 失败

### DISCUSSION THREAD

**Discussion by zhuzeyu22** [midwayjs/midway#3847] [Q&A]
### Describe the problem（描述问题）

我在 bootstrap 里面引入了 Jaeger 并且只想在开发环境使用，且是一体化项目。
我尝试     "dev": "cross-env NODE_ENV=local mwtsc --watch --run @midwayjs/mock/app.js", 
改成  "dev": "cross-env NODE_ENV=local NODE_DEBUG=midway:debug mwtsc --watch --run ./bootstrap.js", 
但是启动失败了，求帮助！

### Midway Versions（Midway 版本）

midway-version@1.3.8

**Comment by czy88840616** [comment]
什么失败了？错误是什么？

**Reply by zhuzeyu22** [reply]
找不到 bootstrap 模块

**Reply by zhuzeyu22** [reply]
我用的是 nvs 做 node 版本管理

**Reply by czy88840616** [reply]
那装一下就好了吧..

**Comment by zhuzeyu22** [comment]


**Reply by zhuzeyu22** [reply]
引入的是根目录下的 bootstrap.js 这个文件呀，已经有的呀，但是不知道为啥 require 就会报错

**Reply by czy88840616** [reply]
截个图？
